### PR TITLE
Feature/aos 2187

### PR DESCRIPTION
**Pull request by bjartek**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -15,15 +15,13 @@ def systemtest = [
...
 ]
 
-
 def overrides = [
   affiliation: "paas",
   testStages:[systemtest],
   piTests: false,
-  // We had to disable sonar integration because Jenkins terminated the report generation and failed the build
-  // completely. There is a Jira task (https://aurora/jira/browse/AOS-1951) to investigate the issue.
-  sonarQube: false,
```
Comment (arevatte) [inline Jenkinsfile]: If SonarQube works again, please enable it.

File: src/test/groovy/no/skatteetaten/aurora/openshift/reference/springboot/controllers/ExampleControllerTest.groovy [hunk 2]
```diff
@@ -11,23 +11,18 @@ import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.
...
 import io.micrometer.spring.autoconfigure.MetricsAutoConfiguration
-import io.micrometer.spring.autoconfigure.export.StringToDurationConverter
 import no.skatteetaten.aurora.AuroraMetrics
-import no.skatteetaten.aurora.annotations.AuroraApplication
 
 @SpringBootTest(classes = [Config, RestTemplate, MetricsAutoConfiguration, AuroraMetrics], webEnvironment = NONE)
 class ExampleControllerTest extends AbstractControllerTest {
 
-  //TODO: The following two lines can be removed once next rc of micrometer is out.
   @Configuration
```
Comment (arevatte) [inline src/test/groovy/no/skatteetaten/aurora/openshift/reference/springboot/controllers/ExampleControllerTest.groovy]: Why isn't this line removed as well?

File: Jenkinsfile (hunk 1 above)
Comment (bjartek) [reply Jenkinsfile]: But it is enabled by default. I removed the disabling of it.

Comment (arevatte) [reply Jenkinsfile]: Sorry. My bad

**Pull request merged**